centos.org/jobs/katello-pipelines.yml

```diff
@@ -8,7 +8,6 @@
       - '4.1'
     distro:
       - centos7
-      - centos8
```

Comment:
I'm not sure we should drop this. I'm leaning to first a commit that moves everything from centos8 to centos8 stream, then a commit that adds 4.4.